### PR TITLE
Fixes the AI from being able to magically turn on broken APCs

### LINKED ISSUE #35257: Cyborgs can control-click broken APCs back online
1. Break an APC, power to the area is disabled as expected.
2. Control click broken APC as a cyborg (probably AI too), power to the area is restored.

Local server, ce50d105

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 

:cl: Dax Dupont
fix: AIs can no longer turn on broken APCs,
/:cl:

[why]: Fixes #35257

### CODE REVIEW

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -912,6 +912,8 @@
 	return 1
 
 /obj/machinery/power/apc/proc/toggle_breaker()
+	if((stat & (BROKEN|MAINT)) || failure_timer)
```
Comment (Cyberboss) [inline code/modules/power/apc.dm]: use is_operational() instead of BROKEN, check and see if it check MAINT and failure_timer too

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by DaxDupont** [comment]
@Cyberboss done

File: code/modules/power/apc.dm [hunk 2]
```diff
@@ -912,7 +912,7 @@
 	return 1
 
 /obj/machinery/power/apc/proc/toggle_breaker()
-	if((stat & (BROKEN|MAINT)) || failure_timer)
+	if(!is_operational() || (stat & MAINT) || failure_timer)
```
Comment (Cyberboss) [inline code/modules/power/apc.dm]: https://github.com/tgstation/tgstation/blob/36b6b72b0268bab04198f3bd7ec6543e3772260a/code/game/machinery/_machinery.dm#L207-L208

MAINT isn't necessary

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by DaxDupont** [comment]
i fix

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**